hardware/opentrons_hardware/hardware_control/tool_sensors.py

```diff
@@ -85,11 +88,14 @@ async def capacitive_pass(
     mover: NodeId,
     distance: float,
     speed: float,
+    sensor_id: Optional[SensorId] = SensorId.S0,
```

Comment:
This looks like a redundant `Optional`.

```suggestion
    sensor_id: SensorId = SensorId.S0,
```